public/components/common/helpers/ppl_docs/commands/rename.ts

```diff
@@ -7,22 +7,19 @@ export const renameCmd = `## rename
 ---
 ### Description
 
-Using \`rename\` command to rename one or more fields in the search
-result.
+Use the \`rename\` command to rename one or more fields in the search result.
 
 ### Syntax
 
 rename &lt;source-field&gt; AS &lt;target-field&gt;\[","
 &lt;source-field&gt; AS &lt;target-field&gt;\]...
 
--   source-field: mandatory. The name of the field you want to rename.
--   field list: mandatory. The name you want to rename to.
+-   \`source-field\`: Required. The field to rename.
+-   \`field list\`: Required. The field's new name.
```

Comment:
Global comment: Should this be field**-**list?

Reply:
@pjfitzgibbons Should `field list` be field**-**list? Or should it be target-field, as in the syntax above?

Reply:
it should be "target-field" as in the example.